docs/generate_switch_config.md

```diff
@@ -75,7 +75,7 @@ ntp enable
 | `--shcd`              | SHCD File                                                                  |
 | `--tabs`              | The tabs on the SHCD file to check, e.g. 10G_25G_40G_100G,NMN,HMN.         |
 | `--corners`           | The corners on each tab, comma separated e.g. 'J37,U227,J15,T47,J20,U167'. |
-| `--csi_folder`        | Directory containing the CSI json file                                     |
+| `--sls_file`        | File containing the CSI json file                                     |
```

Comment:
not sure if the formatting is just wrong on github but this should be formatted to match the other lines